mw-webapp/src/component/progressBar/ProgressBar.module.scss

```diff
@@ -19,7 +20,11 @@ $progressBarBorderRadius: 20px;
 }
 
 .progressIndicatorPercent {
+  position: absolute;
+  right: 0;
+  left: 0;
```

Comment:
why not to handle positioning with flexboxes? Flexbox css has pretty good interface

Reply:
so why not to use flexboxes?